test/fixtures/key/expected.js

```diff
@@ -1,4 +1,10 @@
 "use strict";
 
-elementOpen("div", "1");
+elementOpen("div");
+elementOpen("div", "1", ["key", "1"])
+elementClose("div")
```

Comment:
These need to be expression statements, `elementClose(...)elementOpen(...)` will break.